test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonLight/SingletonBatchRoundtripJsonLightTests.cs

```diff
@@ -729,6 +797,53 @@ private enum BatchFormat
     ]
 }";
 
+        private const string ExpectedReferenceUriRequestPayload_UseDifferentBaseUri = @"
+{
+    ""requests"": [{
+            ""id"": ""1"",
+            ""atomicityGroup"": ""11d431dd-cfee-48c8-95fb-da8491644fa6"",
+            ""method"": ""PUT"",
+            ""url"": ""http://odata.org/test//MySingleton"",

```

Comment:
>// [](start = 44, length = 2)

Why do all of these tests use double-forward-slashes? Looks like a typo that got propagated throughout all of the tests...

Reply:
Looks like the source of the issue is in methods like CreateBatchRequestWithChangesetFirstAndQueryLast that create the request URI by appending a relative url starting with "/" to the servicedocumenturi which terminates in the "/". We should fix both the methods that create the requests as well as the expected output.

---
In reply to: [197298533](https://github.com/OData/odata.net/pull/1195#discussion_r197298533) [](ancestors = 197298533)

Reply:
Fixed. Removed the trialing '/' in sericeDocuemtnUri and updated the expected values.

---
In reply to: [197300176](https://github.com/OData/odata.net/pull/1195#discussion_r197300176) [](ancestors = 197300176,197298533)